README.md

```diff
@@ -34,7 +34,7 @@ You can read about the "classic" confinement policy [here](https://insights.ubun
 
 If you want to use `apt`, then you can add the [juju-golang PPA](https://launchpad.net/~juju/+archive/ubuntu/golang) and then run the following
 
-    sudo apt install golang-1.9
+    sudo apt install golang-1.10
```

Comment:
let's change this to be the instructions for using the snap instead of apt

Reply:
hi @wallyworld  removed apt install doc and updated snap cmd to lock down version to install.